web/styles/modal.css

```diff
@@ -1179,3 +1179,7 @@ ul.modal-options-list {
         background: var(--color-background-active-popover-menu);
     }
 }
+
+#confirm_delete_attachments_modal .detached-attachments-modal-list {
+    margin-bottom: 0;
+}
```

Comment:
I think we should aim towards having comments for overrides like this explaining the reason for the override; with how much cleaner we're getting this file, it'll be really handy to be able to read what the story is with the various overrides, so we can think about having a declarative way to remove them. (I'm not suggesting we add such a way at this time).